test/org/rust/lang/core/parser/RustCTRSTestCase.kt

```diff
@@ -6,16 +6,16 @@ import com.intellij.psi.impl.DebugUtil
 import org.assertj.core.api.Assertions.assertThat
 import java.io.File
 
-public class RustCTRSTestCase : RustParsingTestCaseBase("ctrs") {
+public class RustCTRSTestCase : RustParsingTestCaseBase("ctrs/test") {
 
     fun testCtrs() {
         var nFilesVisited = 0
-        FileUtil.visitFiles(File(myFullDataPath, "test"), {
+        FileUtil.visitFiles(File(myFullDataPath), {
             if (it.isFile && it.extension == myFileExt.trimStart('.')) {
                 nFilesVisited++;
                 val text = FileUtil.loadFile(it, CharsetToolkit.UTF8)
                 val psi = createPsiFile(it.name, text)
-                val expectedError = expectedErrors.contains(it.path)
+                val expectedError = !expectedErrors.find { path -> it.path.endsWith(path) }.isNullOrEmpty()
```

Comment:
Could this be simplified to `expectedErrors.any { path -> it.path.endsWith(path) }` ?

I was wrong when I used `it` in this huge lambda :) May be a nice idea to change `it` to `file` (wow, there is even an intention for it).